apps/api/src/app/bridge/bridge.controller.ts

```diff
@@ -50,13 +50,11 @@ export class BridgeController {
   @Get('/status')
   @UseGuards(UserAuthGuard)
   async health(@UserSession() user: UserSessionData) {
-    const result = await this.getBridgeStatus.execute(
+    return this.getBridgeStatus.execute(
```

Comment:
Note that the [HealthCheck](https://github.com/novuhq/novu/blob/452c5b8411dbc23231266cc62152f06e74250f8c/packages/framework/src/types/health-check.types.ts#L2) type has a status `error` and is not currently used. 

As part of future work, we will discuss whether Bridge errors must return a HealthCheck payload with status set to 'error'.